portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsSelectFromListField.testcase

```diff
@@ -197,14 +197,65 @@ definition {
 	}
 
 	@description = "Verify that options can be sorted alphabetically"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test OptionsCanBeSortedAlphabetically {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-131372 OptionsCanBeSortedAlphabetically pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Select from List");
+
+		FormFields.editOptionsRow(
+			optionName = "Option1",
+			optionValue = "Option A",
+			rowNumber = "1");
+
+		FormFields.editOptionsRow(
+			optionName = "Option2",
+			optionValue = "Option C",
+			rowNumber = "2");
+
+		FormFields.editOptionsRow(
+			optionName = "Option3",
+			optionValue = "Option B",
+			rowNumber = "3");
+
+		Form.gotoAdvancedTab();
+
+		FormViewBuilder.checkSwitcher(propertyName = "alphabeticalOrder");
+
+		Form.save();
+
+		FormsAdminNavigator.gotoPreviewForm();
+
+		Navigator.selectSpecificWindow(windowTitle = "Form - Forms");
+
+		Click(
+			key_fieldName = "Select from List",
+			locator1 = "FormFields#SELECT_FIELD");
+
+		var iterator = "1";
+
+		for (var option : list "A,B,C") {
+			Form.viewSelectFromListOnPreview(
+				fieldName = "Option ${option}",
+				rowPosition = "${iterator}");
+
+			var iterator = MathUtil.sum("${iterator}", "1");
```

Comment:
@ewertonluna, does the reserved word `var` is really needed here? Won't it redeclare the variable `iterator` already declared on line **246**?

If so, can you please fix it?